packages/server/src/stages/rpc/index.ts

```diff
@@ -69,7 +69,7 @@ export default getIsomorphicRpcHandler(
   '${resolverPath}',
   '${resolverName}',
   '${resolverType}',
-) as typeof resolverModule.default
+) ${isTypescript() ? "as typeof resolverModule.default" : ""}
```

Comment:
We are probably going to want to cache isTypeScript outside the hot path as this won't change on the fly - especially as it runs a fs.existsSync call. If we can call it in the creation stage and hold on to the output then that should be fine.

Reply:
An ideal solution would calculate this once and pass this in as config but that is not 100% necessary.

Reply:
Ok, I cached isTypescript on `blitz start` and pass down as config var so it can be used anywhere else in future. Please see my last push.

Reply:
Looks as though there are some tests that also need to be adjusted. I was hesitant about doing this as we should probably refactor the way config is injected here (my fault honestly) but this looks all good if you can get tests to pass.

Reply:
No problem, I'll take a look tomorrow.

Reply:
@ryardley finally it was easy task to get tests passed so I did it immediately. I just added `isTypescript` to mocked stage args inside test. PR is ready to final check and merge I think :-)